README.md

```diff
@@ -80,11 +80,13 @@ csr.public_key = certificate_private_key.public_key
 csr.sign(certificate_private_key, OpenSSL::Digest::SHA256.new)
 
 # We can now request a certificate
-https_cert = client.new_certificate(csr) # => #<OpenSSL::X509::Certificate ....>
+https_cert = client.new_certificate(csr) # => #<Acme::Certificate ....>
```

Comment:
Nitpicking but I wouldn't call it `https_cert` given that its just a certificate that would work any type of tls backed service